pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/SegmentDeletionManager.java

```diff
@@ -241,55 +290,64 @@ protected void removeSegmentFromStore(String tableNameWithType, String segmentId
       String rawTableName = TableNameBuilder.extractRawTableName(tableNameWithType);
       URI fileToDeleteURI = getFileToDeleteURI(rawTableName, segmentId);
       if (fileToDeleteURI == null) {
-        LOGGER.warn("No segment file found for segment: {} in deep store, skipping deletion", segmentId);
```

Comment:
This is redundant. A similar log statement is logged by the `getFileToDeleteURI` method.

```LOGGER.error("No file found for segment: {} in deep store", segmentId);```